torch/nn/functional.py

```diff
@@ -5035,6 +5035,18 @@ def scaled_dot_product_attention(query, key, value, attn_mask=None, dropout_p=0.
     >>> with torch.backends.cuda.sdp_kernel(enable_math=False):
     >>>     F.scaled_dot_product_attention(query,key,value)
 
+    Using Attention Biases:
```

Comment:
You could say that explicitly one of these options is equivalent to setting `is_causal=True`.

Reply:
I'd also consider adding a testing that asserts that